### PR TITLE
feat: Move from signup page to login page

### LINKED ISSUE #2009: Feature: Option to redirect to Login page for existing user from Sign Up page
**Describe the feature you'd like**
There should be an option in the Sign Up page, clicking on which will redirect to Login Page.

**Additional context**
This feature targets those user who already have account in Susi but tries to register again mistakenly. This will allow them to redirect to Login page, and hence they can login from here.

**Screenshots**
login

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #2009 

Changes: 
* LoginActivity.kt
* SignUpActivity.kt
* activity_sign_up.xml
* strings.xml

Screenshots for the change: 
signUpToLogin

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
Is n't the app crashing when entering into settings 
Just asking cause it is crashing for me ? :)

**Comment by atm1504** [comment]
> Is n't the app crashing when entering into settings
> Just asking cause it is crashing for me ? :)

No, it is working fine in my mobile

**Comment by atm1504** [comment]
@iamareebjamal @batbrain7 @arundhati24  please review it

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -156,6 +156,7 @@
     <string name="share_items">Share</string>
 
     <string name="sign_up">Sign Up</string>
+    <string name="signUpToLogin">Existing User?\n Click here to Login</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Strange name. Should be `existing_user_login`

See that every other string is snake case and you have used camel case

Comment (atm1504) [reply app/src/main/res/values/strings.xml]: I will correct my mistake sir soon

**Comment by atm1504** [comment]
@iamareebjamal I have made the requested changes. Please review it now

**Pull request merged**